templates/notes/getting_started.md

```diff
@@ -88,7 +88,7 @@ $$$
 
 ###1.5. Images
 
-Images can be dragged and dropped over the editor. Alternatively, you can use the standard markdown syntax to insert images.
+Images can be dragged and dropped over the editor. Alternatively, you can copy&paste image data (not files) directly to editor (supported only by Chrome, Firefox and Internet Explorer 11). Lastly, you can use the standard markdown syntax to insert images.
```

Comment:
Yeah it should. Basically what the browsers doesn't allow, is to access files on filesystem (which is weird, because they allow it with drag&drop, and paste event also consist of dataTransfer object as the drag&drop events), so you can past only direct image data (open image and copy it inside of the editor), which should be true regarding screenshots in Windows, but I am not running it so can't verify it. And regarding OS X, I have no idea how screenshots works there, so can't say, but if it is similar to Windows behavior than yes, that should also work.